syntax = "proto3";

option java_multiple_files = true;
option java_package = "io.grpc.examples.gfssi";
option java_outer_classname = "GFSSIProto";
option objc_class_prefix = "RTG";

package gfssi;

service DownLoad {
  // Sends a greeting
  rpc DownloadAreaData (DownLoadAreaDataRequest) returns (DownLoadAreaDataResponse) {}
}

message DownLoadAreaDataRequest {
  string date_start = 1;
  string date_end = 2;
  float left_up_lon = 3;
  float left_up_lat = 4;
  float right_down_lon = 5;
  float right_down_lat = 6;
  string frequency = 7;
  string resolution_type = 8;
  string resultid = 9;
}

message DownLoadAreaDataResponse {
  string code = 1;
  string zip_file = 2;
}
